component/table.jsx

```diff
@@ -0,0 +1,48 @@
+import { css } from '../lib/dom.js'
+
+css(`
+.container {
+  overflow-x:auto;
+}
+.customers {
+  font-family: Arial, Helvetica, sans-serif;
+  border-collapse: collapse;
+  width: 100%;
+}
+
+.customers td, .customers th {
+  border: 1px solid #ddd;
+  padding: 8px;
+}
+
+.customers tr:nth-child(even){background-color: #e67e22;}
+
+.customers th {
+  padding-top: 12px;
+  padding-bottom: 12px;
+  text-align: center;
+  background-color: #27ae60;
```

Comment:
we already have a nice way to apply colors to components using pre-defined colors, here is the full list:
```js
[
  "background-darker",
  "background-dark",
  "background",
  "background-light",
  "background-lighter",
  "selection-darker",
  "selection-dark",
  "selection",
  "selection-light",
  "selection-lighter",
  "foreground-darker",
  "foreground-dark",
  "foreground",
  "foreground-light",
  "foreground-lighter",
  "comment-darker",
  "comment-dark",
  "comment",
  "comment-light",
  "comment-lighter",
  "cyan-darker",
  "cyan-dark",
  "cyan",
  "cyan-light",
  "cyan-lighter",
  "green-darker",
  "green-dark",
  "green",
  "green-light",
  "green-lighter",
  "orange-darker",
  "orange-dark",
  "orange",
  "orange-light",
  "orange-lighter",
  "pink-darker",
  "pink-dark",
  "pink",
  "pink-light",
  "pink-lighter",
  "purple-darker",
  "purple-dark",
  "purple",
  "purple-light",
  "purple-lighter",
  "red-darker",
  "red-dark",
  "red",
  "red-light",
  "red-lighter",
  "yellow-darker",
  "yellow-dark",
  "yellow",
  "yellow-light",
  "yellow-lighter"
]
```